### PR TITLE
New version: ErikAraojo.FintX version 1.0.0

### PR DESCRIPTION
**Pull request by namigop**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/134403)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 

(Automated response - build 574.)

**Comment by namigop** [comment]
The default installation mode is non-admin, but via the dialog (as in the screenshot) the user can choose to install in admin mode.  Is this behavior, being able to select between admin vs non-admin, not allowed for winget?

**Comment by namigop** [comment]
Just to add : The installer can be called with the switches` /SILENT /CURRENTUSER` to install silently in non-admin mode or `/SILENT /ALLUSERS` for admin install. for both cases the dialog will not show up and installation will proceed automatically

File: manifests/e/ErikAraojo/FintX/1.0.0/ErikAraojo.FintX.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ErikAraojo.FintX
+PackageVersion: 1.0.0
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/namigop/FintX/releases/download/rel-v1.0/FintXSetup-win-x64.exe
+  InstallerSha256: 20CCE0F3289975C67FB4F86F6F2EE754A135949E20CC63632BC4D21834B12A57
```
Comment (stephengillie) [inline manifests/e/ErikAraojo/FintX/1.0.0/ErikAraojo.FintX.installer.yaml:10]:
```suggestion
  InstallerSha256: 20CCE0F3289975C67FB4F86F6F2EE754A135949E20CC63632BC4D21834B12A57
  Scope: user
  InstallerSwitches:
    Silent:  /SILENT /CURRENTUSER
    SilentWithProgress:  /SILENT /CURRENTUSER
- Architecture: x64
  InstallerUrl: https://github.com/namigop/FintX/releases/download/rel-v1.0/FintXSetup-win-x64.exe
  InstallerSha256: 20CCE0F3289975C67FB4F86F6F2EE754A135949E20CC63632BC4D21834B12A57
  Scope: machine
  InstallerSwitches:
    Silent:  /SILENT /ALLUSERS
    SilentWithProgress:  /SILENT /ALLUSERS
```

**Comment by stephengillie** [comment]
Hi @namigop,

> This PR's version number 1.0.0 has 3 parameters (sets of numbers between dots - major, minor, etc), which is greater than the current manifest's version 0.2, which has 2 parameters.

Is this intentional?

(Automated response - build 623)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**